nerdlets/create-flow/index.js

```diff
@@ -21,7 +21,8 @@ const CreateFlowNerdlet = () => {
   const [page, setPage] = useState('start');
   const [{ accountId }] = usePlatformState();
   const { user } = useFetchUser();
-  const { data: accounts = [] } = useAccountsQuery();
+  const { data: accounts } = useAccountsQuery();
+  const accountName = (accounts.find((acc) => acc.id === accountId) || {}).name;
```

Comment:
This approach is not recommended in the newer versions of React. I'd recommend setting `accountName` using `useState`, and define a `useEffect` with `accounts` and `accountId` as dependencies. Set the `accountName` within that `useEffect`. That way `accountName` only gets set when `accounts` or `accountId` changes.